Terminal.Gui/ConsoleDrivers/WindowsDriver.cs

```diff
@@ -780,15 +780,11 @@ public COLORREF (uint value)
 internal class WindowsDriver : ConsoleDriver {
 	WindowsConsole.ExtendedCharInfo [] _outputBuffer;
 	WindowsConsole.SmallRect _damageRegion;
-	Action<KeyEvent> _keyHandler;
-	Action<KeyEvent> _keyDownHandler;
-	Action<KeyEvent> _keyUpHandler;
-	Action<MouseEvent> _mouseHandler;
 
 	public WindowsConsole WinConsole { get; private set; }
 
 	public override bool SupportsTrueColor => RunningUnitTests || (Environment.OSVersion.Version.Build >= 14931
-		&& (_isWindowsTerminal || _parentProcessName == "devenv"));
```

Comment:
"devenv" is returned when is running from the VS2022. "wininit" isn't ensured to be correct because you can have Windows Terminal open with other thing and not being running `Terminal.Gui`.

Reply:
In my case, "wininit" started being returned when running in the debugger in VS2022. I don't know why, but I never see "devinit" anymore.

Reply:
@tig I already know why I added the "devenv" into the code. If you run UICatalog directly from cmd or conhost, true color is never support because they don't process Ansi escape sequences. But if you run directly from VS2022, no mater Windows Terminal or Console Host is the default terminal, VS2022 VSConsole can always process Ansi escape sequences and thus supporting true color. So, what is needed to differ here is using `_isWindowsTerminal` to allow or not set the alternate screen and using the `_parentProcessName` to allow or not true color.
The "wininit" seems to be unrelated because it's a OS file (C:\Windows\System32\wininit.exe).

Please change to this:
```cs
	public override bool SupportsTrueColor => RunningUnitTests || (Environment.OSVersion.Version.Build >= 14931
		&& (_isWindowsTerminal || _parentProcessName == "wininit" || _parentProcessName == "devenv"));

```